abcorp/src/screens/FlightDestinyScreen.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react';
+import {View, StyleSheet} from 'react-native';
+import {BtnBack} from '../components/booking/BtnBack';
+import {FlightForm} from '../components/booking/FlightForm';
+import {InformationOrigin} from '../components/booking/flightOrigin/InformationOrigin';
+
+export const FlightDestinyScreen = ({navigation, route}) => {
+  const [flightDestiny, setflightDestiny] = useState('');
+  const {flightData} = route.params;
+  return (
+    <View style={styles.mainContainer}>
+      <BtnBack navigation={navigation} />
+      <InformationOrigin flightOrigin={flightData} />
+      <FlightForm
+        txt="Where will you be flying to?"
+        navigation={navigation}
+        flightDestinyScreen="FlightCalendarScreen"
+        flightOrigin={flightDestiny}
+        setflightOrigin={setflightDestiny}
+        flightDataComplete={flightData}
+      />
+    </View>
```

Comment:
this and the origin component are the same, should send some props and use the same, not build quals pages